src/Plugin/License.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace Kirby\Plugin;
+
+use Stringable;
+
+/**
+ * License
+ *
+ * @package   Kirby Plugin
+ * @author    Bastian Allgeier <bastian@getkirby.com>
+ * @link      https://getkirby.com
+ * @copyright Bastian Allgeier
+ * @license   https://getkirby.com/license
+ */
+class License implements Stringable
+{
+	protected LicenseStatus $status;
+
+	public function __construct(
+		protected string $name,
+		protected string|null $link = null,
+		LicenseStatus|null $status = null
+	) {
+		$this->status = $status ?? LicenseStatus::from('unknown');
+	}
+
+	/**
+	 * Returns the string representation of the license
+	 */
+	public function __toString(): string
+	{
+		return $this->name();
+	}
+
+	/**
+	 * Creates a license instance from a given value
+	 */
+	public static function from(self|array|string|null $license): static
```

Comment:
```suggestion
	public static function from(License|array|string|null $license): static
```

More common in the rest of our code